game_eggs/steamcmd_servers/stationeers/stationeers_vanilla/egg-stationeers.json

```diff
@@ -4,50 +4,68 @@
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2022-12-30T11:39:51+01:00",
-    "name": "Stationeers",
-    "author": "eggs@goover.dev",
-    "description": "Stationeers Server",
-    "features": [
-        "steam_disk_space"
-    ],
+    "exported_at": "2024-02-21T04:35:19+01:00",
+    "name": "Stationeer (Vanilla)",
```

Comment:
Wy just not slpit them? 1 wine version from you and a native linux one from Goover.